libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx

```diff
@@ -79,6 +80,30 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
                         action("onOptionsChange");
                     }}
                     openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={true}
+                />
+
+                <div className="library-component">
+                    <h4>EmbedInsightDialogBase without Web component tab</h4>
+                </div>
+
+                <EmbedInsightDialogBase
+                    code={""}
+                    embedTab={"react"}
+                    embedTypeOptions={reactOptions}
+                    propertiesLink={
+                        "https://sdk.gooddata.com/gooddata-ui/docs/area_chart_component.html#properties"
+                    }
+                    integrationDocLink={"https://sdk.gooddata.com/gooddata-ui/docs/platform_integration.html"}
+                    onClose={action("onClose")}
+                    onCopyCode={action("onCopyCode")}
+                    onTabChange={action("onTabChange")}
+                    onOptionsChange={(opts) => {
+                        setReactOptions(opts as IReactOptions);
+                        action("onOptionChange");
+                    }}
+                    openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={false}
```

Comment:
Please move this prop to line 97, as props which are non-function should be next to each other, for the sake of consistency.